components/TripList/TripList.js

```diff
@@ -1,41 +1,96 @@
 import React, { Component } from "react";
-import { View, Text, ScrollView } from "react-native";
+import { Button, Modal, ScrollView, Text, View } from "react-native";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import { getTrips, deactivateTrip } from "../../util/apiCalls";
+import { setTrips } from "../../actions";
 import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
 import styles from "./styles";
 
 export class TripList extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      activeTripId: 0,
```

Comment:
I don't believe you're using activeTripId anymore? So maybe just remove it from state, since you're using the id in the Redux store.